docs/framework-jrebel_agent.md

```diff
@@ -1,31 +1,32 @@
 # JRebel Agent Framework
-The JRebel Agent Framework causes an application to be automatically configured to work with an IDE using [JRebel][].
+
+This framework enables the use of [JRebel][jrebel] with deployed applications. Pushing any [JRebel Cloud/Remote][remoting] enabled application (containing `rebel-remote.xml`) will automatically download the latest version of [JRebel][jrebel] and set it up for use.
 
 <table>
   <tr>
     <td><strong>Detection Criterion</strong></td>
-    <td>Existence of the `rebel.xml` and `rebel-remote.xml` files in either the root or `WEB-INF/classes` directory or the application.</td>
+    <td>Presence of a <tt>rebel-remote.xml</tt> file inside the application archive. This file is present in every application that is configured to use <a href="http://manuals.zeroturnaround.com/jrebel/remoting/index.html" target="_blank">JRebel Cloud/Remote</a>.</td>
   </tr>
   <tr>
     <td><strong>Tags</strong></td>
     <td><tt>jrebel-agent=&lt;version&gt;</tt></td>
   </tr>
 </table>
-Tags are printed to standard output by the buildpack detect script
+
+For more information regarding setup and configuration, please refer to the [JRebel with Pivotal Cloud Foundry tutorial][pivotal].
 
 ## Configuration
-For general information on configuring the buildpack, refer to [Configuration and Extension][].
 
-The framework can be configured by modifying the [`config/jrebel_agent.yml`][] file in the buildpack fork.  The framework uses the [`Repository` utility support][repositories] and so it supports the [version syntax][] defined there.
+The framework can be configured by modifying the [`config/jrebelagent.yml`][jrebelagentyml] file in a buildpack fork. The framework uses the [`Repository` utility support][repositories], supporting the [version syntax][] defined there.
 
 | Name | Description
 | ---- | -----------
 | `repository_root` | The URL of the JRebel repository index ([details][repositories]).
-| `version` | The version of Jrebel to use. Candidate versions can be found in [this listing][].
+| `version` | The version of JRebel to use.
```

Comment:
Candidate versions should continue to be linked, but the listing simply needs to be updated to point to your index instead of ours.